SETUP WORKING ENVIRONMENT.md

```diff
@@ -0,0 +1,39 @@
+# HOW TO SETUP WORKING ENVIRONMENT ON YOUR LOCAL MACHINE
+
+## To setup Backend using Nodejs-
+- ***Firstly install Node on your machine, [click here](https://nodejs.org/en/).***
+-***To check that Node has successfully installed, run the following command in Terminal. This will display the version of Node installed.***
+
+```js
```

Comment:
It's shell code, not JS code.

**```js**  :x: 

**```bash** ✅

Reply:
> It's shell code, not JS code.
> 
> **```js** 
> 
> **```bash**

I added 
```js 
to just highlight the section. I had'nt added any JS to the code.
If you still want me to change the code, I can do that ??

Reply:
I had changed the code ...
Review it please